content/en/docs/ambient/architecture/_index.md

```diff
@@ -18,18 +18,6 @@ To enforce L7 policies, add the `istio.io/use-waypoint` label to your resource t
   precedence over the namespace waypoint as long as the service waypoint can handle service or all traffic.
   Similarly, a label on a pod will take precedence over a namespace label
 
```

Comment:
We should either link to `usage/add-workloads` here or remove all the specific waypoint label stuff.

Reply:
This section needs to move to add "add workloads" page (and the below to an "add policy" page) once we decide where to put them.  See Slack for bikeshed thread